ql/src/java/org/apache/hadoop/hive/ql/ddl/database/create/CreateDatabaseAnalyzer.java

```diff
@@ -103,6 +104,7 @@ public void analyzeInternal(ASTNode root) throws SemanticException {
       if (managedLocationUri != null) {
         database.setManagedLocationUri(managedLocationUri);
       }
+      database.setOwnerName(SessionState.getUserFromAuthenticator());
```

Comment:
`CreateDatabaseOperation` sets the ownername both for native and remote db type. I wonder, should we follow the same pattern?

Reply:
Thanks for the review @InvisibleProgrammer, actually I was under some other assumption.

I will address this.